docs/fsharp/tutorials/fsharp-interactive/index.md

```diff
@@ -13,102 +13,233 @@ To run F# Interactive from the console, run `dotnet fsi`. You will find `dotnet
 
 For information about command line options available, see [F# Interactive Options](../../language-reference/fsharp-interactive-options.md).
 
-To run F# Interactive through Visual Studio, you can click the appropriate toolbar button labeled **F# Interactive**, or use the keys **Ctrl+Alt+F**. Doing this will open the interactive window, a tool window running an F# Interactive session. You can also select some code that you want to run in the interactive window and hit the key combination **Alt+Enter**. F# Interactive starts in a tool window labeled **F# Interactive**. When you use this key combination, make sure that the editor window has the focus.
+## Executing code directly in F# Interactive
 
-Whether you are using the console or Visual Studio, a command prompt appears and the interpreter awaits your input. You can enter code just as you would in a code file. To compile and execute the code, enter two semicolons (**;;**) to terminate a line or several lines of input.
+Because F# Interactive is a REPL, you can execute code interactively in it. Here is an example of an interactive session after executing `dotnet fsi` from the command line:
```

Comment:
Probably good to define REPL on first use:

```suggestion
Because F# Interactive is a REPL (read-eval-print loop), you can execute code interactively in it. Here is an example of an interactive session after executing `dotnet fsi` from the command line:
```